### PR TITLE
Moved management commands to commands app and updated travis config

### PR DESCRIPTION
**Pull request by Samy-33**
Run:  
* `$ pipenv run ./manage.py run_tests` to run tests
* `$ pipenv run ./manage.py run_coverage` to generate coverage report

### CODE REVIEW

File: commands/management/commands/run_coverage.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import sys
+
+
+class Command(BaseCommand):
+    help = 'Runs coverage'
+
+    def handle(self, *args, **kwargs):
+        if os.getenv('PY_ENV') != 'test':
+            sys.exit('Please ensure PY_ENV environment variable is set to \'test\'')
+        os.system('pipenv run ./manage.py test -v 2 --noinput\
```
Comment (sk364) [inline commands/management/commands/run_coverage.py:12]: we need `call_command` here.

File: commands/management/commands/run_tests.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+import sys
+
+
+class Command(BaseCommand):
+    help = 'Runs tests'
+
+    def handle(self, *args, **kwargs):
+        if os.getenv('PY_ENV') != 'test':
+            sys.exit('Please ensure PY_ENV environment variable is set to \'test\'')
+        os.system('pipenv run ./manage.py test -v 2 --noinput')
```
Comment (sk364) [inline commands/management/commands/run_tests.py:12]: and here!

**Review by sk364 — CHANGES_REQUESTED** [review]

File: commands/management/commands/run_coverage.py (hunk 1 above)
Comment (Samy-33) [reply commands/management/commands/run_coverage.py:12]: Can't do that. Please refer to this [issue](https://github.com/jazzband/django-nose/issues/256) in `django-nose`.  
For the time being we can can just use `os.system`.

Comment (sk364) [reply commands/management/commands/run_coverage.py:12]: Interesting.

**Review by sk364 — DISMISSED** [review]

**Review by sk364 — DISMISSED** [review]

**Review by sk364 — APPROVED** [review]

**Pull request merged**